sdk/connectedvmwarevsphere/Azure.ResourceManager.ConnectedVMwarevSphere/tests/tests/Tests/ClusterTests.cs

```diff
@@ -0,0 +1,162 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
+using Azure.ResourceManager.TestFramework;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using Azure.Core;
+using System.Linq;
+using Azure.ResourceManager.ConnectedVMwarevSphere.Models;
+using Azure.ResourceManager.ConnectedVMwarevSphere.Tests.Helpers;
+
+namespace Azure.ResourceManager.ConnectedVMwarevSphere.Tests.tests.Tests
+{
+    public class ClusterTests : ConnectedVMwareTestBase
+    {
+        public ClusterTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        private async Task<VMwareClusterCollection> GetVMwareClusterCollectionAsync()
+        {
+            var resourceGroup = await CreateResourceGroupAsync();
+            return resourceGroup.GetVMwareClusters();
+        }
+
+        [AsyncOnly]
+        [TestCase]
+        [RecordedTest]
+        public async Task CreateDelete()
+        {
+            var clusterName = Recording.GenerateAssetName("testcluster");
+            var _clusterCollection = await GetVMwareClusterCollectionAsync();
+            var _extendedLocation = new ExtendedLocation()
+            {
+                Name = CustomLocationId,
+                Type = EXTENDED_LOCATION_TYPE
+            };
+            var clusterBody = new VMwareClusterData(DefaultLocation);
+            clusterBody.MoRefId = "domain-c7";
+            clusterBody.VCenterId = VcenterId;
+            clusterBody.ExtendedLocation = _extendedLocation;
+            // create cluster
+            VMwareCluster cluster1 = (await _clusterCollection.CreateOrUpdateAsync(clusterName, clusterBody)).Value;
```

Comment:
This method is `CreateDelete`, but we only test `Create` here.

Reply:
Actually there is not any method expose by sdk to delete the resource, but the delete functionality is implicitly taken by delete resource group, since after every run resource group is getting deleted hence the created resources implicitly getting deleted.